tools/build-a-node

```diff
@@ -184,20 +185,22 @@ if [[ -n ${MAKE_JOBS} ]]; then
 fi
 make ${JOBS_ARG} V=1 CC=$GCC CXX=$GXX \
      CFLAGS="${cflags}" CXXFLAGS="${cflags}"
-make V=1 CC=$GCC CXX=$GXX install
+make V=1 CC=$GCC CXX=$GXX \
+     CFLAGS="${cflags}" CXXFLAGS="${cflags}" install
 
 # Optionally include GCC runtime libs. If a version of GCC was used to compile
 # node is greater than the version of the GCC runtime libs included in the
 # platform *and* this node build is targetted for use in the GZ, then the
 # node build needs to (a) include the needed GCC runtime libs and (b) set a
 # RUNPATH for the node binary to find them.
 #
-# See <https://github.com/joyent/rfd/tree/master/rfd/0059#problems-with-the-gz>
+# See <https://github.com/TritonDataCenter/rfd/tree/master/rfd/0059#problems-with-the-gz>
 # for details.
 includeGccRuntimeLibs=$(echo "$buildOpts" | json include_gcc_runtime_libs)
 if [[ -n "$includeGccRuntimeLibs" ]]; then
     ldd ./node  | awk '{print $3}' | grep '^/opt/local/gcc' | while read lib; do
         cp $lib $buildDir/node/lib/
+        $ELFEDIT -e 'dyn:runpath $ORIGIN' $buildDir/node/lib/$(basename $lib)
```

Comment:
Why do we need this?  I'm sure there's a good reason, but I just can't immediately figure it out.

Reply:
It fixes the rpath of `libstdc++.so.6` which itself links against libgcc_s. Without this `ldd` of node shows libgcc_s coming from both the `node/lib` directory (its own dependency) as well as `/opt/local` (the indirect dependency via libstdc++).